README.md

```diff
@@ -1 +1,161 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics-app <a name="about-project"></a>
+
+> this is Metrics app project i have created using  React/redux
+
+## 🛠 Built With <a name="built-with"></a>
+
+1: React
+2: CSS
+3: linters
+4: Redux
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://React.dev/">React</a></li>
+    <li><a href="https://Redux.com/">Redux toolkit</a></li>
+  </ul>
+</details>
+</details>
+
+<!-- Features -->
+<p> the project explanation </p> 
+
+### Key Features <a name="key-features"></a>
+
+
+-A metrics web app designed to display country flags, populations, and currencies provides users with a convenient and visually engaging way to access key information about countries around the world.
+
+### Testing <a name="Testing"></a>
+
+- coming soon
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1: web browser
+2: an code editor
+3: node installed on your pc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd to your desired folder 
+  git clone https://github.com/MohamedRoshdy2021/metrics-app.git
+### Install
+
+Install this project with:
+  install node_modules by this command : npm i 
+
+then:
+
+run this command: npm run start 
+
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Roshdy**
+- GitHub: [@MohamedRoshdy2021](https://github.com/MohamedRoshdy2021)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-elkhadragy-2b58b6215/)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- will enhance the design in the next branch 
+- will make the version of the mobile more responsive for better UI to the user   
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project don't forgt to give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microvesre who helped us during our journey
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Please ensure that the "Acknowledgements" section gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).